.github/workflows/conformance-aws-cni.yaml

```diff
@@ -199,6 +199,7 @@ jobs:
           region: ${{ matrix.region }}
           owner: "${{ steps.vars.outputs.owner }}"
           version: ${{ matrix.version }}
+          spot: ${{ github.event_name == 'schedule' }}
```

Comment:
Isn't this doing the opposite of what we want? 🤔

Reply:
omg yes :facepalm:
fixed, thanks.